### DISCUSSION TITLE
Scalability and performance with multiple instances

### DISCUSSION THREAD

**Discussion by lumos309** [codesandbox/codesandbox-client#8167] [Q&A] [answered]
Hey all,

I've been considering using codesandbox as a pilot tool for research - basically we'd direct research participants to our sandbox to use some interface and complete tasks. I was wondering how capable the platform is of handling multiple users (e.g., 10-100) opening the same sandbox at once? Is there a way of monitoring server resource usage to see how close we are to capping out?

**Comment by JamesACS** [comment, marked as answer]
Hello @lumos309 ,

Our set up should be able to comfortably handle over 100 users accessing the same environment. Unfortunately we don't have a way of monitoring this right now. But if you do encounter issues please let us know and we'll investigate.

**Reply by lumos309** [reply]
Awesome! Thanks a lot :)
